Sources/SwiftGRPCNIO/HTTP1ToRawGRPCClientCodec.swift

```diff
@@ -15,6 +15,7 @@
  */
 import Foundation
 import NIO
+import NIOTLS
```

Comment:
Can this be removed?

Reply:
Not sure what this is doing here in the first place!